src/functions.php

```diff
@@ -17,6 +17,7 @@
 
 use Cake\ORM\Table;
 use Cake\ORM\TableRegistry;
+use Cake\Routing\Router;
```

Comment:
Umm..why?

Reply:
Cause I'm dumb. I was thinking of adding a url() function and then changed my mind.

Reply:
I have at times contemplated having a `url()` function in core itself :)